src/go/rpk/pkg/cli/cmd/wasm/common.go

```diff
@@ -60,7 +60,7 @@ func ExistingTopic(admin sarama.ClusterAdmin, topic string) (bool, error) {
 }
 
 func CreateDeployMsg(
-	name string, description string, content []byte,
+	name string, coproc_type string, description string, content []byte,
```

Comment:
Please use camel-case for variable and function names in the Go code 🙏🏽

Reply:
Done